wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -37,8 +38,12 @@
 )
 from wemake_python_styleguide.visitors.base import BaseTokenVisitor
 
-EMPTY_TOKEN = tokenize.TokenInfo(
-    type=0, string='', start=(0, 0), end=(0, 0), line='',
+SENTINEL_TOKEN: Final = tokenize.TokenInfo(
+    type=ENDMARKER,
+    string='',
+    start=(ENDMARKER, ENDMARKER),
+    end=(ENDMARKER, ENDMARKER),
```

Comment:
let's use `0` here because as far I know it's for row and column, so `ENDMARKER` usage is quite confusing here

Reply:
Opps, I misunderstood your last comment. Fixed now.